notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Since it's an open source ai cookbook, we should stick to open source models. Can you use a model from Hugging Face Hub?  E.g. mistral?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>For the generation, I think indeed we can use Mixtral.</p><p> </p><p>For the LLM-as-a-judge part, however, the OS models I tried so far were not at all satisfactory. I tried Prometheus-13B, JudgeLM-33B, and Mixtral: when I asked them to evaluate answers and matched their score to my own evaluation, none of them come close to GPT4's accuracy. If you have other suggestions, I can try!</p>

Reply:
<p>How should we serve Mixtral, via an endpoint ?</p>

Reply:
re: serving, sure, an endpoint will work. 
It's a shame that other LLMs don't come close. Maybe folks on HF Slack can recommend a model? If not, I suppose it's ok to mix and match. one model for generation, one model as a judge.